charts/trino/values.yaml

```diff
@@ -197,3 +197,38 @@ worker:
     # requests:
     #   cpu: 100m
     #   memory: 128Mi
+
+schemas: {}
+  # Custom schemas that will be mounted in /etc/trino/schemas
```

Comment:
This still requires a cluster restart.

And Kafka is not the only connector which uses table definition files. IMO connector specific functionality should not be part of the main chart - even moreso since we don't have connector specific tests (and adding them would require a lot of work and maintenance).

I'd like to hear opinion of @radek-starburst , @przemekak about this.

Reply:
Whats the alternative way to create those schemas file?
Because I am kind of OK with this change if it makes life easier for the user.
I am not concernad that much about tests here, as they are just testing if trino is deployed correctly not if it works corectly afterwards, and to be honest I dont think such detailed tests should be a part of helm tests.

Reply:
Let's rename the key to something like `kafka_table_descriptions` then? That makes it easy to find docs on Trino side for it as well. The only alternative to adding such files is to either bake it into the container image (not a good idea) or to fork the chart (not a good idea).

For someone who's new to the chart they might feel lost when searching about `schemas`.